### PR TITLE
Add LegalEntityID to GetEmployeesRaw and GetProjectsRaw

### PR DESCRIPTION
**Pull request by lwschan**

### CODE REVIEW

File: TimeLog.Api.Documentation/Source/Reporting/GetEmployeesRaw.xsd [hunk 1]
```diff
@@ -42,6 +42,7 @@
       <xsd:element name="EmployeeNo" type="xsd:string" minOccurs="0" maxOccurs="1" />
       <xsd:element name="Username" type="xsd:string" minOccurs="0" maxOccurs="1" />
       <xsd:element name="CostPrice" type="xsd:decimal" minOccurs="0" maxOccurs="1" />
+      <xsd:element name="LegalEntityID" type="xsd:positiveInteger" />
```
Comment (oexenhave) [inline TimeLog.Api.Documentation/Source/Reporting/GetEmployeesRaw.xsd]: Please add `minOccurs` and `maxOccurs` for completeness.

File: TimeLog.Api.Documentation/Source/Reporting/GetProjectsRaw.xsd [hunk 2]
```diff
@@ -67,6 +67,7 @@
       <xsd:element name="CurrencyISOCode" type="xsd:string" />
       <xsd:element name="CurrencyRate" type="xsd:decimal" />
       <xsd:element name="IsMultiCustomerProject" type="xsd:boolean" />
+      <xsd:element name="LegalEntityID" type="xsd:positiveInteger" />
```
Comment (oexenhave) [inline TimeLog.Api.Documentation/Source/Reporting/GetProjectsRaw.xsd]: Please add `minOccurs` and `maxOccurs` for completeness.

**Review by oexenhave — CHANGES_REQUESTED** [review]

File: TimeLog.Api.Documentation/Source/Reporting/GetProjectsRaw.xsd (hunk 2 above)
Comment (lwschan) [reply TimeLog.Api.Documentation/Source/Reporting/GetProjectsRaw.xsd]: It's added.

File: TimeLog.Api.Documentation/Source/Reporting/GetEmployeesRaw.xsd (hunk 1 above)
Comment (lwschan) [reply TimeLog.Api.Documentation/Source/Reporting/GetEmployeesRaw.xsd]: It's added.

**Review by oexenhave — APPROVED** [review]
Looks good

**Pull request merged**